proto/parca/query/v1alpha1/query.proto

```diff
@@ -256,10 +259,55 @@ message QueryRequest {
   optional SourceReference source_reference = 9;
 
   // which runtime frames to filter out, often interpreter frames like python or ruby are not super useful by default
-  optional RuntimeFilter runtime_filter = 10;
+  optional RuntimeFilter runtime_filter = 10 [deprecated = true];
 
   // invert_call_stack inverts the call stacks in the flamegraph
   optional bool invert_call_stack = 11;
+
+  // a varying set of filters to apply to the query request
+  Filters filters = 12;
```

Comment:
Oh apologies for missing this earlier, this should be repeated, it should be possible to set multiple filters.